speechbrain/lobes/models/transformer/Transformer.py

```diff
@@ -518,6 +533,7 @@ def __init__(
         attention_type="regularMHA",
         ffn_type="regularFFN",
         ffn_cnn_kernel_size_list=[3, 3],
+        output_hidden_states=False,
```

Comment:
layerdrop is missing here now though :)

Reply:
(and in the code beneath)

Reply:
Why you trolling me :'( It is not! Just not at the same place ! Look a few lines above.

Reply:
oops